robolectric/src/main/java/org/robolectric/junit/rules/BackgroundTestRule.java

```diff
@@ -19,20 +19,20 @@
  *
  * <p>Example usage:
  *
- * <pre>{@code
- * @Rule public final BackgroundTestRule backgroundTestRule = new BackgroundTestRule();
+ * <pre>
+ * &#064;Rule public final BackgroundTestRule backgroundTestRule = new BackgroundTestRule();
  *
- * @Test
- * @BackgroundTest
+ * &#064;Test
```

Comment:
I don't think it's a great idea to use unicode number to replace at symbol. It will increase the difficulty for Robolectric developers to read code.